pyston/utils/files.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+from six import BytesIO
+
+from django.core.exceptions import RequestDataTooBig
+
+from pyston.conf import settings
+
+
+def get_file_content_from_url(url, timeout=1):
+    resp = requests.get(url, timeout=timeout, stream=True)
+
+    content = b''
```

Comment:
proč to vlastně nedáváš rovnou do toho BytesIO? Případně rovnou neplníš InMemoryUploadedFile